src/AcmTable/AcmTable.tsx

```diff
@@ -720,6 +745,14 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
                                     /* istanbul ignore next */
                                     rows.length && (bulkActions?.length || !!props.onSelect) ? onSelect : undefined
                                 }
+                                onCollapse={
+                                    props.subItems?.length
```

Comment:
This is ugly; but when you add the `onCollapse` the PF table will reserve a `th` in the Table; which off sets the index count on the header rows, which ends up messing up the sorting behavior =/